beyond-essence/inventory/modal-dialogs.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Modal dialogs
+background_image: index.jpg
+---
+
+Modal dialogs display new content to the user without taking them to a different view.
+Users need to interact with the modal dialog before continuing with further tasks.
+
+## Use cases
+
+In the Beyond cockpit we provide two types of modal dialogs:
+
```

Comment:
Again I struggled a bit about the sentence structure but as long as the following sentence is quite short and then it is kind of fine but maybe we wanna change this in the future ( because there was the issue with the dropdown- but it ´s up to you :-) )